pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -363,7 +362,7 @@ export class SETroubleshootPage extends React.Component {
                 ];
                 // if the alert has level "red", it's critical
                 const criticalAlert = (itm.details && 'level' in itm.details && itm.details.level == "red")
-                    ? <ExclamationTriangleIcon className="ct-icon-exclamation-triangle" size="md" />
+                    ? <ExclamationTriangleIcon className="ct-icon-exclamation-triangle pf-c-icon pf-m-lg" />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-130726-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)